offchain-modules/packages/x/src/handlers/eth.ts

```diff
@@ -488,3 +489,7 @@ export function parseLockLog(log: Log, parsedLog: ParsedLog): ParsedLockLog {
     asset,
   };
 }
+
+function toCKBAddress2021(address: string): string {
+  return generateAddress(parseAddress(address));
```

Comment:
should be refactored after the new release of lumos

Reply:
delete short id of lumos config to workaround